cli/src/semgrep/commands/ci.py

```diff
@@ -563,7 +564,7 @@ def ci(
 
     logger.info("CI scan completed successfully.")
     logger.info(
-        f"  Found {unit_str(num_blocking_findings + num_nonblocking_findings, 'finding')} ({num_blocking_findings} blocking) from {unit_str(num_executed_rules, 'rule')}."
+        f"  Found {unit_str(num_blocking_findings + num_nonblocking_findings, 'finding')} ({num_blocking_findings} blocking) from {unit_str(len(filtered_rules), 'rule')}."
```

Comment:
Judging by the updated snapshots, it looks like this number isn't right, and is larger that num_executed_rules. Not sure why though